sources/api/early-boot-config/src/provider/vmware.rs

```diff
@@ -190,21 +183,22 @@ mod error {
         #[snafu(display("Unable to read input file '{}': {}", path.display(), source))]
         InputFileRead { path: PathBuf, source: io::Error },
 
-        #[snafu(display("Error serializing TOML to JSON: {}", source))]
-        SettingsToJSON { source: serde_json::error::Error },
+        #[snafu(display(
+            "Invalid (non-utf8) output from base64 string '{}': {}",
+            base64_string,
+            source
+        ))]
+        InvalidUTF8 {
```

Comment:
Looks like this is being added in the wrong commit; it's not used until the guestinfo commit.

Reply:
Fixed